components/admin_console/cloud_migrate/migrate_workspace.tsx

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import AlertBanner from 'components/alert_banner';
+import BlockableLink from 'components/admin_console/blockable_link';
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import AdminHeader from 'components/widgets/admin_console/admin_header';
+import CloudCard from 'components/common/cloud_card';
+import {useInterval} from 'components/common/hooks/set_interval';
+
+import {
+    CloudLinks,
+} from 'utils/constants';
+
+import './migrate_workspace.scss';
+
+import Badge from 'components/widgets/badges/badge';
+
+import ProgressBar from './widgets/progress_bar';
+
+import importsListTable from './components/import_list';
```

Comment:
Can we move this `components` all together in the above block to keep the imports organized?

Reply:
it is a different components folder. The initial structure for the import module looks like:
-containerComponent 
    - components 
    - widgets
    
    
<img width="519" alt="Captura de pantalla 2021-02-24 a las 19 21 46" src="https://user-images.githubusercontent.com/10082627/109047298-faa28a80-76d5-11eb-875d-d0ea67c74cb3.png">